recipes/libtiff/all/conanfile.py

```diff
@@ -120,7 +122,27 @@ def _patch_sources(self):
         tools.replace_in_file(cmakefile,
                               "add_subdirectory(tools)\nadd_subdirectory(test)\nadd_subdirectory(contrib)\nadd_subdirectory(build)\n"
                               "add_subdirectory(man)\nadd_subdirectory(html)", "")
-        tools.replace_in_file(cmakefile, "LIBLZMA_LIBRARIES", "LibLZMA_LIBRARIES")
+        if tools.Version(self.version) < "4.3.0":
+            tools.replace_in_file(cmakefile, "LIBLZMA_LIBRARIES", "LibLZMA_LIBRARIES")
```

Comment:
Not required anymore, `xz_utils` recipe has been fixed in https://github.com/conan-io/conan-center-index/pull/5231

Reply:
```suggestion
```